openbook_common/utils/rq_helpers.py

```diff
@@ -0,0 +1,60 @@
+import django_rq
+from django_rq.utils import get_statistics, FailedJobRegistry
+
+
+class RQStats():
+
+    def __init__(self):
+
+        self.stats = get_statistics()
+        self.queue_name = 'default'
```

Comment:
hmm 🤔 django_rq uses `get_statistics` for their `manage.py rqstats` command. This leads me to believe that `self.stats` will have a non-dynamic version of the statistics (a snapshot if you will) when the `RQStats` class is initialised. 

Since the helper methods use the queue objects obtained from `self.stats`,  can you double check if the job account is indeed the "live" count and not the one returned by `get_statistics` when the class is initialised.

Reply:
Yes, but it is not an issue in this case. The methods are only used once, and all at the same time. There is no delay or incorrect display of statistics. But thanks, good point!